src/Beman/Optional26/tests/test_types.hpp

```diff
@@ -4,7 +4,9 @@
 #ifndef BEMAN_OPTIONAL26_TESTS_TEST_TYPES_HPP
 #define BEMAN_OPTIONAL26_TESTS_TEST_TYPES_HPP
 
-namespace beman::optional26::test {
+#include <compare>
+
+namespace beman::optional26::tests {
```

Comment:
why longer name? :D

Reply:
Matching the namespaces to the path name and include guard. It's not really important here, but an internal linter checks that canonical names of headers matches the namespaces used so that collisions and ambiguities are avoided.

Reply:
cool. let's keep it